catalog/pages/inputs/index.md

```diff
@@ -690,11 +690,12 @@ span: 6
     <Container>
         <Row >
             <Column medium={4}>
-                <DropDownLabel size="small">Label text</DropDownLabel>
+                <DropDownLabel id="labelText" size="small">Label text</DropDownLabel>
```

Comment:
Always favor lowercase, for elements, for attributes and their values, for classes and ids. Multi-word names for classes and ids should either 1., concatenate the words in lowercase without any in-between character, or 2., separate each word with a "-" (not "_") and maintain lowercasing throughout.

Reply:
updated